src/common/connection.js

```diff
@@ -148,10 +148,12 @@ class Connection extends EventEmitter {
       cert: this._certificate
     }, _.isUndefined);
     const websocketOptions = _.assign({}, options, optionsOverrides);
-    const websocket = new WebSocket(this._url, websocketOptions);
+    const websocket = new WebSocket(this._url, null, websocketOptions);
```

Comment:
Not sure that we can call this a bug. Native `WebSocket` constructor only has two arguments, and `ws` package allows to skip that second argument.